### PR TITLE
Add tests for evaluation with initialization in TF

### PR DESCRIPTION
**Pull request by negvet**
### Changes

Add tests for evaluation with initialization in TF.

FQ with multiple inbound nodes initialization fix

### Reason for changes

Run sanity tests on initialization.

Not able to register FQ layer stats

### Related tickets

58869, 75958

### CODE REVIEW

**Comment by negvet** [comment]
Jenkins please retry a build

**Review by alexsu52 — CHANGES_REQUESTED** [review]
Please add tests (ref 58869)

**Comment by negvet** [comment]
TODO:
- [ ] remove commented code (required for local run)

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by pfinashx — APPROVED** [review]

File: tests/tensorflow/sota_checkpoints_eval.json [hunk 1]
```diff
@@ -131,6 +138,7 @@
                     "config": "examples/tensorflow/classification/configs/quantization/mobilenet_v3_small_imagenet_int8.json",
                     "reference": "mobilenet_v3_small",
                     "target": 67.75,
+                    "target_init": 0.1,
```
[the remark this answers is not in the document]
Comment (negvet) [reply tests/tensorflow/sota_checkpoints_eval.json:141]: Thanks for the notice. Indeed it is very low. We have a separate ticket for this issue. I propose to run these tests anyway just for sanity perposes.

File: tests/tensorflow/test_init.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+DIFF_TARGET_INIT_MAX_GLOBAL = 1.0
+
+MODE = 'TF2'
+
+
+class TestInitialization(RunTest):
+    @staticmethod
+    def update_tag_text(tag, text):
+        with tag('p'):
+            with tag('span', style='Background-color: #{}'.format(BG_COLOR_GREEN_HEX)):
```
[the remark this answers is not in the document]
Comment (negvet) [reply tests/tensorflow/test_init.py]: Thanks for the comment. Updated!

**Pull request merged**